tests/test_complex.py

```diff
@@ -72,3 +74,9 @@ def test_map_lambda_as_element():
     """Test that a map lambda is held as a single element"""
     stack = run_vyxal("⁽ƛ1+;M", inputs=[[[1, 2], [3, 4]]])
     assert stack[-1] == [[2, 3], [4, 5]]
+
+
+def test_vectorise_map_lambda():
```

Comment:
Maybe add tests for filter/sort as well?

Reply:
Good idea but i is 2 lzy